src/prelude/internals.mo

```diff
@@ -390,8 +390,25 @@ let @ic00 = actor "aaaaa-aa" :
     } -> async ()
  };
 
+func @ic00_create_canister() : shared {
+      settings : ?@ManagementCanister.canister_settings
+    } -> async { canister_id : Principal } {
+  @ic00.create_canister
+};
+
+func @ic00_install_code() : shared {
+    mode : { #install; #reinstall; #upgrade };
+    canister_id : Principal;
+    wasm_module : @ManagementCanister.wasm_module;
+    arg : Blob;
+  } -> async () {
+  @ic00.install_code
+};
+
 // It would be desirable if create_actor_helper can be defined
 // without paying the extra self-remote-call-cost
+// TODO: This helper is now only used by Prim.createActor and could be removed, except
```

Comment:
Does it have a clean substitute? If so, reply to the forum and remove this guy. (In a separate PR/issue).

Reply:
https://github.com/dfinity/motoko/issues/3420